backend/googlephotos/googlephotos.go

```diff
@@ -52,8 +52,8 @@ const (
 	listChunks                  = 100 // chunk size to read directory listings
 	albumChunks                 = 50  // chunk size to read album listings
 	minSleep                    = 10 * time.Millisecond
-	scopeReadOnly               = "https://www.googleapis.com/auth/photoslibrary.readonly"
-	scopeReadWrite              = "https://www.googleapis.com/auth/photoslibrary"
+	scopeReadOnly               = "https://www.googleapis.com/auth/photoslibrary.appendonly"
+	scopeReadWrite              = "https://www.googleapis.com/auth/photoslibrary.appendonly"
```

Comment:
Shouldn't it be
```go
scopeReadOnly               = "https://www.googleapis.com/auth/photoslibrary.readonly.appcreateddata"
scopeReadWrite              = "https://www.googleapis.com/auth/photoslibrary.readonly.appcreateddata https://www.googleapis.com/auth/photoslibrary.appendonly"
```
?

They separated the permissions so that read and write are separate permissions. There's no "ReadWrite" permission anymore. To sync photos and videos, both are though. Also, `scopeReadOnly` should definitely point to `readonly.appcreateddata` (as `appendonly` isn't a read permission).

See https://developers.google.com/photos/overview/authorization#library-api-scopes

Your fix didn't work for me, but mine did.

Did sync work for you with your fix?

Reply:
I've incorporated that into a new version and I've tested read only and read write and all seems to be working now.